plugin/runmetrics/producer.go

```diff
@@ -219,6 +229,52 @@ func newMemStats(producer *producer) (*memStats, error) {
 		return nil, err
 	}
 
+	// GC
+	memStats.gcSys, err = producer.createInt64GaugeEntry("process/gc_sys", "Bytes of memory in garbage collection metadats", metricdata.UnitBytes)
+	if err != nil {
+		return nil, err
+	}
+
+	memStats.otherSys, err = producer.createInt64GaugeEntry("process/other_sys", "Bytes of memory in miscellaneous off-heap runtime allocations", metricdata.UnitBytes)
+	if err != nil {
+		return nil, err
+	}
+
+	memStats.numGC, err = producer.createInt64GaugeEntry("process/num_gc", "Cumulative count of completed GC cycles", metricdata.UnitDimensionless)
+	if err != nil {
+		return nil, err
+	}
+
+	memStats.numForcedGC, err = producer.createInt64GaugeEntry("process/num_forced_gc", "Cumulative count of GC cycles forced by the application", metricdata.UnitDimensionless)
+	if err != nil {
+		return nil, err
+	}
+
+	memStats.nextGC, err = producer.createInt64GaugeEntry("process/next_gc", "Target heap size of the next GC cycle in bytes", metricdata.UnitBytes)
+	if err != nil {
+		return nil, err
+	}
+
+	memStats.lastGC, err = producer.createInt64GaugeEntry("process/last_gc", "Time the last garbage collection finished, as milliseconds since 1970 (the UNIX epoch)", metricdata.UnitMilliseconds)
```

Comment:
next_gc and last_gc appears to be similar but quite different. Perhaps the could be renamed to

next_gc_heap_size
last_gc_finished_timestamp